launcher.py

```diff
@@ -399,6 +405,26 @@ def initialise_watcher_postgres_database(self) -> bool:
         )
         return False
 
+    def update_watcher_postgres_database(self) -> bool:
```

Comment:
Running it locally I can see `ecto.reset` command logs, but can't find `ecto.migrate`'s one. I'd expect to see smth like:
`2019-03-22 12:24:42.824 [info] module=Ecto.Migrator function=log/2 ⋅Already up⋅`

We'll see...